packages/web-components/src/components/ic-stepper/ic-stepper.stories.mdx

```diff
@@ -95,3 +95,120 @@ import Step from "../ic-step/readme.md";
     </ic-stepper>`}
   </Story>
 </Canvas>
+
+### Compact stepper
```

Comment:
When I click on the buttons to change the current step on the compact stepper, the first time I click through them, the step-type doesn’t appear e.g. see the last step below - and then if I click back through them (so click Back and then Next again), it shows ‘Completed’

![image](https://user-images.githubusercontent.com/111882109/210615255-7c2e119c-5c79-4cdd-a600-107210b7155e.png)

Not sure if this was happening before your latest updates - sorry if it was and I missed it! Maybe it’s just an issue with the `<script>` but thought it best to point it out :) I think everything else looks great though!

Reply:
This is as expected. All steps by default start off active unless the user states otherwise. In the `<script>`, I have an array that works by changing the `stepType` of the compact steps after their first time as the current step. This is to replicate the behaviour/logic a user may build into the stepper if they're using a form with the stepper for example. 

With the default stepper, `stepType="current"` has its own styling. However, we don't want this to be the case with the compact stepper. In the code for the stepper, I've built the compact stepper to remember this `stepType` of the step before it was current so that this styling is still displayed. This also prevents a breaking change.

(@GCHQ-Developer-847, @MI6-451)